src/sentry/features/manager.py

```diff
@@ -97,4 +102,61 @@ def _get_handler(self, feature, actor):
             rv = handler(feature, actor)
             if rv is not None:
                 return rv
-        return None
+            return None
+
+    def has_for_organization(self, name, organization, objects, actor=None):
```

Comment:
Having reflected on it a bit, the phrase "for organization" is misleading. I meant it to indicate that we can do a check that logically cares only about the organization. But it's likelier to be taken (wrongly) to mean that you need to call this instead of `has` if the flag is organization-scoped. (Similarly, `OrganizationFeatureHandler` does not necessarily handle an `OrganizationFeature`.)

I've been using the word "batch" below and a few other places. I think in the morning (PDT) I'll change `*_for_organization` everywhere to `*_for_batch`. I'm open to other suggestions.

Reply:
`has_for_batch` or `has_bulk` both make more sense to me. I was confused but this name and what it was going to do when I encountered it earlier.